lib/Sema/CSClosure.cpp

```diff
@@ -530,18 +530,13 @@ class SyntacticElementConstraintGenerator
       init = TypeChecker::buildDefaultInitializer(patternType);
     }
 
-    if (init) {
-      return SolutionApplicationTarget::forInitialization(
-          init, patternBinding->getDeclContext(), patternType, patternBinding,
-          index,
-          /*bindPatternVarsOneWay=*/false);
-    }
-
-    // If there was no initializer, there could be one from a property
-    // wrapper which has to be pre-checked before use. This is not a
-    // problem in top-level code because pattern bindings go through
-    // `typeCheckExpression` which does pre-check automatically and
-    // result builders do not allow declaring local wrapped variables.
+    // A property wrapper initializer (either user-defined
+    // for a synthesized one) has to be pre-checked before use.
```

Comment:
Looks like there's a small typo:
```suggestion
    // or a synthesized one) has to be pre-checked before use.
```

Reply:
Thanks, fixed!